Content.Server/Botany/Systems/PlantAnalyzerSystem.cs

```diff
@@ -0,0 +1,261 @@
+using Content.Server.Botany.Components;
+using Content.Server.PowerCell;
+using Content.Shared.DoAfter;
+using Content.Shared.Interaction;
+using Content.Shared.PlantAnalyzer;
+using Robust.Server.GameObjects;
+using Robust.Shared.Audio.Systems;
+using Robust.Shared.Player;
+using Robust.Shared.Prototypes;
+using System.Linq;
+using System.Threading;
+
+namespace Content.Server.Botany.Systems;
+
+public sealed class PlantAnalyzerSystem : EntitySystem
+{
+    [Dependency] private readonly PowerCellSystem _cell = default!;
+    [Dependency] private readonly SharedAudioSystem _audio = default!;
+    [Dependency] private readonly SharedDoAfterSystem _doAfterSystem = default!;
+    [Dependency] private readonly UserInterfaceSystem _uiSystem = default!;
+    [Dependency] private readonly IPrototypeManager _prototypeManager = default!;
+
+    private Dictionary<EntityUid, CancellationTokenWrapper> _cancellationTokenSources = new Dictionary<EntityUid, CancellationTokenWrapper>();
+
```

Comment:
Don't do this.
1. It doesn't serialize properly (serialization attribute does not make it serialize).
2. Systems shouldn't be storing stateful data.
3. It's unneeded.